tests/test/faaslet/test_threads.cpp

```diff
@@ -53,14 +32,4 @@ TEST_CASE("Run thread checks locally", "[threads]")
 {
     runTestLocally("threads_check");
 }
-
-TEST_CASE("Run thread checks with chaining", "[threads]")
-{
-    runTestDistributed("threads_check");
-}
-
-TEST_CASE("Run distributed threading check", "[threads]")
-{
-    runTestDistributed("threads_dist");
-}
```

Comment:
These two functions weren't really testing distributed execution, they were just doing a _lot_ of fiddling with internals to fake up what we think distributed execution will do. Instead I'll leave this to another PR on doing "proper" distributed testing with multiple containers.